src/services/auth/userPermissions.go

```diff
@@ -0,0 +1,79 @@
+package auth
+
+import (
+	"github.com/Nastya-Kruglikova/cool_tasks/src/database"
+	"github.com/Nastya-Kruglikova/cool_tasks/src/models"
+	"github.com/Nastya-Kruglikova/cool_tasks/src/services/common"
+	"net/http"
+)
+
+var SpecialPermissions map[string][]string
+
+func init() {
+	SpecialPermissions = make(map[string][]string)
+	SpecialPermissions["/v1/users"] = []string{"admin"}
+}
+
+var CheckPermission = func(userSession string, requiredRole string, itemOwner string) bool {
+	switch requiredRole {
+	case "owner":
+		return isOwner(userSession, itemOwner)
```

Comment:
If you write isOwner (userSession, itemOwner) || isAdmin (session), code will be more readable.

Reply:
It's different logic

Reply:
It's different logic

Reply:
Get. Ok